taskt/Core/Automation/Commands/Data Commands/TextExtractionCommand.cs

```diff
@@ -1,88 +1,83 @@
 ﻿using System;
-using System.Linq;
-using System.Xml.Serialization;
+using System.Collections.Generic;
 using System.Data;
+using System.Drawing;
+using System.Linq;
 using System.Text.RegularExpressions;
-using System.Collections.Generic;
 using System.Windows.Forms;
-using taskt.UI.Forms;
-using taskt.UI.CustomControls;
-using System.Drawing;
+using System.Xml.Serialization;
+using taskt.Core.Automation.Attributes.ClassAttributes;
+using taskt.Core.Automation.Attributes.PropertyAttributes;
 using taskt.Core.Utilities.CommonUtilities;
+using taskt.UI.CustomControls;
+using taskt.UI.Forms;
 
 namespace taskt.Core.Automation.Commands
 {
     [Serializable]
     [Attributes.ClassAttributes.Group("Data Commands")]
-    [Attributes.ClassAttributes.Description("This command allows you to perform advanced string extraction.")]
-    [Attributes.ClassAttributes.UsesDescription("Use this command when you want to extract a piece of text from a larger text or variable")]
-    [Attributes.ClassAttributes.ImplementationDescription("This command implements String.Substring Action on input text or variable to return substring.")]
+    [Description("This command performs advanced text extraction.")]
     public class TextExtractionCommand : ScriptCommand
     {
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Supply the value or variable requiring extraction (ex. [vSomeVariable])")]
-        [Attributes.PropertyAttributes.PropertyUIHelper(Attributes.PropertyAttributes.PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
-        [Attributes.PropertyAttributes.InputSpecification("Select or provide a variable or text value")]
-        [Attributes.PropertyAttributes.SampleUsage("**Hello** or **vSomeVariable**")]
-        [Attributes.PropertyAttributes.Remarks("")]
-        public string v_InputValue { get; set; }
+        [PropertyDescription("Text Data")]
+        [InputSpecification("Provide a variable or text value.")]
+        [SampleUsage("Sample text to perform text extraction on || {vTextData}")]
+        [Remarks("Input data other than string (text) type will cause error.")]
```

Comment:
Change to "Providing data of a type other than a 'String' will result in an error."

Reply:
done